corehq/apps/app_manager/templates/app_manager/profile.xml

```diff
@@ -3,6 +3,7 @@
          update="{{ profile_url }}&amp;profile={{ build_profile_id }}"
          requiredMajor="{{ app.build_spec.major_release }}"
          requiredMinor="{{ app.build_spec.minor_release.1 }}"
+         requiredMinimal="{{ app.build_spec.patch_release.2 }}"
```

Comment:
why is it called "app.build_spec.patch_release.2"? What's the process behind this naming?

Reply:
If you look here https://github.com/dimagi/commcare-hq/pull/27883/files#diff-a65d3e32d1001ad60472a36e6d006b88R230
the function `minor_release` returns two arguments, `return int(major), int(minor)` so value of minor will be in index 1.
So, I followed the same suite of having `patch_release` to return all Semver values and the patch release version will be on index 2.

Reply:
Cool, looks like a JSON object thing to refer by index I think.